assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -162,7 +162,6 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 						this.handleKeyboardNavigation( event );
 						break;
 					case 'Enter':
-					case 'Space':
```

Comment:
Why? `Space` should act like `Enter.
For example: 
https://www.w3.org/WAI/WCAG21/Techniques/client-side-script/SCR35#:~:text=but%20it%20also%20occurs%20when%20the%20user%20focuses%20the%20element%20and%20hits%20enter%20or%20space%2C

Reply:
Product request, look at ED.